clans/src/main/java/me/mykindos/betterpvp/clans/clans/listeners/ClanEventListener.java

```diff
@@ -128,6 +128,11 @@ public void onChunkUnclaim(ChunkUnclaimEvent event) {
         clanManager.getRepository().deleteClanTerritory(targetClan, chunkString);
         targetClan.getTerritory().removeIf(territory -> territory.getChunk().equals(UtilWorld.chunkToFile(chunk)));
 
+        if (targetClan.getHome().getChunk().equals(chunk)) {
+            targetClan.setHome(null);
```

Comment:
You'l want to do this one too


            Block block = clan.getHome().clone().subtract(0, 0.6, 0).getBlock();
            if (block.getType() == Material.RED_BED) {
                block.setType(Material.AIR);
            }

Reply:
good point

Reply:
fixed